_variables/ListsClassesUA.js

```diff
@@ -737,43 +737,39 @@ ClassSubList["war magic"] = {
       name : "Arcane Deflection",
       source : ["UA:WR", 2],
       minlevel : 2,
-      description : "\n   " + "When you are hit by an attack or fail a Constitution saving throw, you can use your reaction" +
-                    "\n   " + "to gain a +2 bonus to your AC against that attack or a +4 bonus to that saving throw." +
-                    "\n   " + "When you use this feature, you can’t cast spells other than cantrips until" +
-                    "\n   " + "the end of your next turn.",
+      description : desc(["When hit by an attack, I can use my reaction to gain a +2 bonus to my AC against that attack",
+                          "When I fail a Con saving throw, I can use my reaction to gain +4 bonus to that saving throw",
+                          "When I use this feature, I can’t cast spells other than cantrips until the end of my next turn"]),
       action: ["reaction", ""]
     },
     "subclassfeature2.1" : {
       name : "Tactical Wit",
       source : ["UA:WR", 2],
       minlevel : 2,
-      description : "\n   " + "You gain a bonus to your initiative rolls equal to your Intelligence modifier",
-      changeeval : "if (classes.known.wizard.level >= 2) {Value(\"Init Bonus\", What(\"Int Mod\"))}"
+      description : desc(["I gain a bonus to my initiative rolls equal to my Intelligence modifier"]),
+      eval : "if (!What(\"Init Bonus\")) {Value(\"Init Bonus\", \"Int\")}",
```

Comment:
I tried to model this off of the Rakish Audacity.  Not sure if it's a bug or not.  But when the Int value changes, it doesn't update the Initiative.
To replicate:
Set Dex to 16
Set Int to 18.
Initiative is +7
If Int is changed to 20, Initiative doesn't change to +8.  
If Int is changed again, say to 21.  Initiative is changed to +8

It changes it when Dex is changed immediately though.